### ISSUE TITLE
Field Suggestions for the Game's Name

### ISSUE THREAD

**Issue by Master-Foo** [Master-Foo/game#3]

**Comment by jrslagle** [comment]
I vote Harvest Mars. But is that too close to Harvest Moon?

**Comment by Master-Foo** [comment]
I wont shoot it down, but I think we can do better.

**Comment by jrslagle** [comment]
4th Rock

Iron Age

Iron Garden

The Martians

**Comment by Master-Foo** [comment]
I think we shouldn't reference Mars specifically.

The reason is, it is trivial to build a system which could represent ANY planet. Mars is just one of those possibilities. We could easily generate a scenario for Europa, Tatooine, Uranus, MYANUS.

Let's not paint ourselves into a corner just because of a name.

**Comment by techwulf** [comment]
Terra
Short for Terraform.

**Comment by jrslagle** [comment]
Desert Rose

**Comment by jrslagle** [comment]
@Master-Foo, I agree that the world parameters should be swappable for any world. But I'm pretty excited about settling / terraforming Mars in the near future. Actually I don't care which rock it is, so long as we can terraform the bitch. (mining, climate/weather modeling, open trade with floating prices, etc)

**Comment by jrslagle** [comment]
@techwolf, Terra alone means Earth.

**Comment by Master-Foo** [comment]
I think Terra<something> is a good word to think about. It describes what the game is, at least in a general sense. So long as we aren't implying "Terra" means Earth specifically.

**Comment by Master-Foo** [comment]
@jrslagle I'm not saying that "Terraforming Mars" isn't one of the goals of this project. What I'm saying is by the time we have implemented the Mars scenario, we will have already long implemented a generic way to procedurally generate a random planet.

So, from a practical perspective, this is not a Mars game. It is a "Terraform in General" game with a Mars specific scenario sometime in the future.

**Comment by Master-Foo** [comment]
Hell, after we get the procedural generation done, we could even have an "Earth-like Planet with Damn-Dirty-Apes and Mother-Fucking Dinosaurs Chasing You" senario.

**Comment by jrslagle** [comment]
Some other ideas from the house

Terra-Marza

Terraform: Iron Lily

![image](https://cloud.githubusercontent.com/assets/11183143/20910785/fb9be3ca-bb20-11e6-827d-196a81d3d27c.png)

**Comment by jrslagle** [comment]
Tom, Alex, and I like TerraCraft.

**Comment by Master-Foo** [comment]
That's the best suggestion I've heard so far. Let's think of way to distinguish that it's Terra **form** as opposed to Terra (Earth).

Also, I've been playing around with the idea of pulling inspiration from the titles of Golden Age Sci-Fi Pulp Fiction novels. As if this game is an Epic story as opposed to just an activity.

**Comment by Master-Foo** [comment]
So, seemingly for no particular reason, I happened to recall Bismarck's famous speech and It occurred to me that the situation he was speaking about is in many ways parallel to what the goals of this game are. Germany at the time was a new nation aggressively expanding into it's new colonies. Expanding it's industrial infrastructure. It's military capabilities. It's scientific capabilities. etc.

Now imagine that happening on Mars.

So... How about... **"Iron and Blood"**

Or, "Iron and Blood: The search for Spock" (Because Star Trek III was about Terraforming)

**Comment by fishscene** [comment]
I'm partial to TerraCraft as well, although I'm seeing a few things related to Minecraft-type deals where the name is used.

We seem to like "Terra", so:
TerraPrep
AmeTerra (Ame: Japanese for "rain" - which might be a critical step in the TerraForming process)
TerraProc(ess)
TerraStage
TerraTech(nique)
TerraCourse
TerraPlan
TerraDev(elopment)
TerraSys(tem)

Note: I haven't researched these out for usage.

**Comment by jrslagle** [comment]
I really like Iron and Blood. I've been having a hard time reconciling making a game enjoyable and Mars/(any cold rock) being so cold and barren. The answer lies somewhere in incorporating that brutality into the game's brand/theme. I would caution that even games that are really hard and unwelcoming in the body of the game, start off welcoming to draw you in (Half-life 2, Skyrim).

**Comment by jrslagle** [comment]
On the foreign language theme, 

I think mixing multiple foreign languages is weird... But the Latin word for rain is vernal. And Japanese for Earth is chikyu and craft/workmanship is zaiku. So Ame-zaiku (雨細工) ?

But I do agree that rain is a critical step in the terraforming process.

What about RainCraft? 

Nova Prospect - an old name I've used for things.  I just like the sound of it and prospects being where you dig a trial mine.

**Comment by jrslagle** [comment]
And some usage research:

TerraCraft - company that makes reverse tricycles. 

Iron and Blood - "Blood and Iron" is the title of a Prussian speech from 1862. Namely "Not through speeches and majority decisions will the great questions of the day be decided ... but by iron and blood."

RainCraft - Nothing major. a MineCraft texture pack. A small forum. A gutter company. An irrigation company. A WoW guild. 

TerraTech - a game on valve where you battle across alien worlds. So definite competition. 

Nova Prospect - Just Nova Prospekt, a Combine-occupied prison within Half-life 2.

**Comment by Master-Foo** [comment]
I'm going to explain my thought process behind disliking the XyzCraft. I think it is derivative and alludes too much toward MineCraft. Really, this game has little to do with MineCraft. The way I see it is, there are more parallels between this game and Eve Online. So, calling this XyzCraft would just send people the message that this is just another "Activity". Another MineCraft knockoff.

Which gets me back to my point I made before about drawing inspiration from the old Golden Age Pulp Fiction stories and such. I think the title should convey that this is an epic (like Eve Online) as opposed to an activity (like MineCraft)

Maybe one example of a name that I really like is the series "The Expanse". It's genius and simple and awesome all at the same time without being derivative.

The title shouldn't describe the activity. It should describe the story we want to play.

**Comment by Master-Foo** [comment]
TLDR; think of it this way... If you were to write a Sci-Fi novel about the shit you want to do in this game, would you call it "TerraCraft"?

**Comment by Master-Foo** [comment]
By the way I'm not shooting down XyzCraft as ideas we shouldn't be exploring. I just think those words should be **concepts** that inspire a title. Not the title itself.

**Comment by jrslagle** [comment]
How about The Thaw. 

-for the tide of humanity melting the icey worlds of the solar system and beyond. It speaks of the era of terraforming. While it reminds you of the first buds of spring, it also acknowledges these worlds start off frozen and barren. It focuses your attention on the long term goal of warming this place up. 

And it doesn't name Mars.

**Comment by Master-Foo** [comment]
This goes against my "Don't mention Mars" rule. But I think it's good enough that I can live with myself as a hypocrite.

"Life on Mars"
